atlas/AtlasMl/tests/conftest.py

```
@@ -49,4 +49,16 @@ def mock_get(key, default=None):
     
     # Cleanup after tests
     os.environ.pop("TESTING", None)
-    
\ No newline at end of file
+    
+
+@pytest.fixture(autouse=True)
+def mock_generate_embeddings_openai():
+    with patch("atlasml.ml.VectorEmbeddings.MainEmbeddingModel.generate_embeddings_openai", return_value=[0.1, 0.2, 0.3]), \
```

Ruff (0.12.2)

Line too long (124 > 88)

(E501)